package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "serve-static",
   "description": "Serve static files",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

```suggestion
  "version": "3.0.0",
```
@wesleytodd @Phillip9587 then we consider this as a major release?

---

I'm in favor of holding off on the release for now. The change is pretty minor, and I haven't checked if there are other issues stacked up that might call for a major.

---

I am also +1 to remove the breaking change from this release and avoid a major

---

If we remove that change we don't have any code changes i think.

---

Hm, @Phillip9587 can you elaborate on how #190 is breaking? I can see a case to be made that it can break clients if the application was setting these headers and they were being overridden. Is that what you are talking about?

If so, then yeah I think we have to back that out and wait for 3.x which would *only be able to ship with `express@6` because this is a pass through. So a breaking change here would be breaking for `express`.

---

I guess also if that is the case for the breaking classification, then we could land a deprecation warning now that the behavior they are using is not really following the *intent* and will break in an upcoming release. In theory we could make this an option, but that comes with some other problems.

The more I think about it the more I think you are right and that this is breaking and needs to be reverted.

---

Yep, it's a breaking change, and +1 on adding the warning.

---

#199